### PR TITLE
Revert token hashing

### PR DESCRIPTION
**Pull request by rmweir**

### CODE REVIEW

File: package/Dockerfile [hunk 1]
```diff
@@ -109,7 +109,7 @@ RUN curl -sLf ${!TINI_URL} > /usr/bin/tini && \
 
 ENV CATTLE_UI_VERSION 2.5.2-rc8.1
 ENV CATTLE_DASHBOARD_UI_VERSION v2.5.2-rc5
-ENV CATTLE_CLI_VERSION v2.4.11-rc1
+ENV CATTLE_CLI_VERSION v2.4.11-rc2
```
Comment (cbron) [inline package/Dockerfile:112]: This is the reverted version right ?

**Review by cbron — APPROVED** [review]
1 question, if that is right then we are good.

File: package/Dockerfile (hunk 1 above)
Comment (rmweir) [reply package/Dockerfile:112]: Yep

**Comment by rmweir** [comment]
Revert due to https://github.com/rancher/rancher/issues/30978, plan on fixing that issue in the future and pulling this back in.

**Comment by ibuildthecloud** [comment]
I'd rather this not be reverted.  I'm going to fix rancher-operator for this.

**Pull request closed without merging**